tests/Utf8Json.Tests/PrittyPrintTest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Utf8Json.Tests
+{
+    public class PrittyPrintTest
+    {
+
+        [Fact]
```

Comment:
I modified to test to add more test cases dor `Double`:

```
        [Theory]
        [InlineData(double.MaxValue, "1.7976931348623157E+308")]
        [InlineData(double.MinValue, "-1.7976931348623157E+308")]
        [InlineData(double.NaN, "NaN")]
        [InlineData(double.NegativeInfinity, "-Infinity")]
        [InlineData(double.PositiveInfinity, "Infinity")]
        public void Double(double value, string expected)
        {
            var bytes = JsonSerializer.Serialize(value);
            var printed = JsonSerializer.PrettyPrint(bytes);

            printed.Is(expected);
        }
```

However NaN and Infinity fails, because `JsonReader.GetCurrentJsonToken` returns `JsonToken.None` in `JsonSerializer.WritePrittyPrint` instead of `JsonToken.Number`. Is that expected?